_security/configuration/demo-configuration.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Demo Configuration Setup Guide
```

Comment:
Can we change this title to "Setting up a demo configuration" or something similar? That way it is consistent with the other items in the group under 'Configuration'.

Reply:
sounds good. I have kept your suggestion.